Examples/cChocoFeatureExample.ps1

```diff
@@ -0,0 +1,28 @@
+
+
+configuration ChocoFeatures {
+
+    Import-DscResource -ModuleName cChoco
+
+    Node 'localhost' {
+
+        cChocoFeature allowGlobalConfirmation {
+
+            FeatureName = "allowGlobalConfirmation"
```

Comment:
So you can't get to the resource name?

The parameter key would be queried if empty is what I'm suggesting. It's okay to go with this as is for now. I'm just thinking of a nice way to optimize.